common/conf/conf.cpp

```diff
@@ -160,6 +160,7 @@ error::Error MenderConfig::ProcessCmdlineArgs(const vector<string> &args) {
 		} else if ((opt_val.option == "--log-level") || (opt_val.option == "-l")) {
 			log_level = opt_val.value;
 		}
+		ex_opt_val = opts_iter.Next();
```

Comment:
Nitpick: Unrelated to "feat: Add MenderContext", should have been in the earlier commit which introduced this loop.

Reply:
Good catch!